### DISCUSSION TITLE
[FR] Québec, Canada VAT = 2 taxes

### DISCUSSION THREAD

**Discussion by kbftech** [baimard/gestion#419] [Features request]
Au québec, on doit fournir 2 taxes et indiquer nos numéros enregistrés

Ma suggestion serait donc d'avoir la possibilitée d'ajouter un autre poucentage de taxe à celui déjà existant. 

SPÉCIFICITÉE: La seconde taxe doit être calculée à partir du total précédent, incluant la première taxe.

*********************************
*** ENGLISH TRANSLATION ***
*********************************

In Québec, we must charge and provide on the bill 2 separate taxes. Each company has it's own tax number associated which must also be provided on the bill.

My suggestion is to add another field for the other tax percentage, and one field per tax to specify the associated number.

NOTE: The second TAX is based on the total amount, including the first tax. We're taxed on the tax...

**********************
*** SUGGESTION ***
**********************

Suggestion "simple" Front-end:

(Numeric) VAT 1 %: 
(Numeric) VAT 1 associated number (leave empty for none):
(Numeric) VAT 2 % (if applicable):
(Alphanumeric) VAT 2 associated number (leave empty for none):
(Checkbox) Second VAT calculated from total including first VAT:

Calcul back-end:
```
Montant_VAT1= (sous-total * VAT1)
Montant_VAT2= ((sous-total + Montant_VAT1) * VAT2)
```

**Comment by baimard** [comment]
Bonjour, c'est pris en compte, pourriez vous me faire une représentation visuel de ce que vous imagineriez vouloir en rendu graphique ?

**Comment by kbftech** [comment]
C'est vraiment pas ma tasse de thé le desing mais bon, j'imagine que qqc comme ça ferait l'affaire:
![image](https://user-images.githubusercontent.com/77502706/199572917-b78f7545-4f1a-4b36-ba33-5b5314596a24.png)

**Comment by kbftech** [comment]
J'ai vu que dans "invoice ninja" par exemple, ils proposent des taxes "inclusives ou exclusives" avec un radio button. J'en comprend que dans certains pays, les taxes sont inclues dans le coût du produit/service et on doit les déduire lors de la facturation alors qu'ici, on doit les ajouter au coût total des services/produits.

Note que pour certains produits/services, les taxes ne sont pas applicables. Ça ne me touche pas, mais je te lance l'info.

Merci!

**Comment by baimard** [comment]
Je vais essayer de m'adapter rapidement merci pour votre retour :)

**Comment by kintaro1981** [comment]
in Italy in some cases we need to add "Marca da bollo" (not in % but a specific amount like 2€ or more)

For example if some invoice/receipt are greater then a specific amount we need to add it.
